sdk/Lusid.Sdk.Tests/tutorials/Ibor/StructuredResultStore.cs

```diff
@@ -0,0 +1,946 @@
+﻿using System;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+
+
+namespace Lusid.Sdk.Tests.tutorials.Ibor
+{
+    public class StructuredResultStore : TutorialBase
+    {
+        // In this example Structured Result Store is used to demonstrate how one can upsert document which contains accruals
+        // but is missing PV's  and have the valuation calculate PV's based on the accruals provided. 
+        [Test]   
```

Comment:
need to add a `[LusidFeature]`, add to https://gitlab.com/finbourne/salesengineering/lusid-features/-/blob/master/features.yml too